cyences_app_for_splunk/default/data/ui/views/cs_overview.xml

```diff
@@ -141,30 +143,12 @@
     <panel depends="$antivirus$">
       <title>Antivirus</title>
       <table id="tbl_antivirus">
-        <search>
-          <query>`cs_cyences_index` alert_name IN ("*Sophos Endpoint Protection*", "*Windows Defender *", "*CrowdStrike *", "*Defender ATP*") | eval cyences_severity = case(cyences_severity=="critical", 5, cyences_severity=="high", 4, cyences_severity=="medium", 3, cyences_severity=="low", 2, cyences_severity=="info", 1) 
-| search cyences_severity IN $tkn_severity$
-| `cs_add_notable_event_editor_fields`
-| search notable_event_status="BANANABANANA" OR notable_event_status IN $tkn_status$
-| stats count as problems by alert_name, cyences_severity | rename alert_name as savedsearch_name
-| join type=left savedsearch_name
-    [| rest /servicesNS/-/cyences_app_for_splunk/saved/searches count=0 splunk_server=local 
-    | fields title action.cyences_notable_event_action.teams 
-    | rename title as savedsearch_name, action.cyences_notable_event_action.teams as teams]
-| append [| rest /servicesNS/-/cyences_app_for_splunk/saved/searches count=0 splunk_server=local | search "eai:acl.app"="cyences_app_for_splunk" "action.cyences_notable_event_action"="1" disabled=0 is_scheduled=1 | fields title action.cyences_notable_event_action.teams
-| rename title as savedsearch_name, action.cyences_notable_event_action.teams as teams | eval cyences_severity="unknown" | `cs_filter_savedsearches("Sophos Endpoint Protection|Windows Defender|CrowdStrike|Defender ATP")`]
-| stats first(*) as * by savedsearch_name, cyences_severity
-| search NOT savedsearch_name IN `cs_monthly_alerts`
-| sort savedsearch_name, cyences_severity
-| streamstats count as newcount by savedsearch_name
-| where (cyences_severity!="unknown" OR newcount=1)
-| makemv delim="," teams
-| eval teams=trim(teams)
-| search $tkn_teams$
-| `cs_filter_savedsearches("Sophos Endpoint Protection|Windows Defender|CrowdStrike|Defender ATP")` 
+        <search base="basesearch">
+          <query>| `cs_filter_savedsearches("Sophos Endpoint Protection|Windows Defender|CrowdStrike|Defender ATP")`
```

Comment:
cs_filter_savedsearches => this macro searching based on the product name so need to change the filters everywhere where we don't have search prefix as a product.

for ex.: line no. 126- we don't have Authentication as a product.

Reply:
same way for "Defender ATP" alerts, product is "Office 365 Defender ATP"

Reply:
Done